rest/service/src/main.rs

```diff
@@ -9,12 +10,19 @@ use rustls::{
 use std::io::BufReader;
 use structopt::StructOpt;
 
+/// uri where the openapi spec is exposed
+pub(crate) const SPEC_URI: &str = "/v0/api/spec";
+
 #[derive(Debug, StructOpt)]
-struct CliArgs {
-    /// The Rest Server address to bind to
+pub(crate) struct CliArgs {
+    /// The Rest https Server address to bind to
     /// Default: 0.0.0.0:8080
-    #[structopt(long, short, default_value = "0.0.0.0:8080")]
-    rest: String,
+    #[structopt(long, long, default_value = "0.0.0.0:8080")]
+    https: String,
+    /// The Rest http Server address to bind to
+    /// Default: 0.0.0.0:8081
+    #[structopt(long, long, default_value = "0.0.0.0:8081")]
```

Comment:
It may be that we wish to not expose HTTP by default, since it's a security threat? Perhaps users should explicitly opt-in?

Reply:
Good catch, I'm not even sure if I want to keep the http there, I'll leave it for now though. I made it an Option. Also need to "fix" https, atm it's just using dummy generated certificates, need to be able to specify them as an option too.